.gitignore

```diff
@@ -3,4 +3,6 @@ dist
 node_modules
 storybook-static
 .vscode
-renovate.json
\ No newline at end of file
+renovate.json
+.idea/
+yarn.lock
```

Comment:
Can you exclude yarn.lock?
We have to track dependency changes!

Reply:
Done !